README.md

```diff
@@ -68,24 +68,25 @@ w3 up recipies.txt
 
 Upload file(s) to web3.storage. The IPFS Content ID (CID) for your files is calculated on your machine, and sent up along with your files. web3.storage makes your content available on the IPFS network
 
-- `--no-wrap` Don't wrap input files with a directory.
-- `-H, --hidden` Include paths that start with ".".
-- `-c, --car` File is a CAR file.
-- `--shard-size` Shard uploads into CAR files of approximately this size in bytes.
-- `--concurrent-requests` Send up to this many CAR shards concurrently.
+* `--no-wrap` Don't wrap input files with a directory.
+* `-H, --hidden` Include paths that start with ".".
+* `-c, --car` File is a CAR file.
+* `--shard-size` Shard uploads into CAR files of approximately this size in bytes.
+* `--concurrent-requests` Send up to this many CAR shards concurrently.
 
 ### `w3 ls`
 
 List all the uploads registered in the current space.
 
-- `--json` Format as newline delimted JSON
-- `--shards` Pretty print with shards in output
+* `--raw` Format full response as JSON
+* `--json` Format as newline delimited JSON
```

Comment:
I've been using `--json` to basically mean the same as `--raw` in other parts of the cli... I think it's ok to include more info in a json representation than we do in the strings for the humans. Would that make sense here too?

Reply:
If this is listing ALL uploads I'd not allow buffering the full response and formatting as JSON.

Reply:
oh yes, you're totally right - `--raw` doesn't make sense in the case where we automatically grab all the pages, will update!